core-web/libs/portlets/dot-analytics/portlet/src/lib/dot-analytics-search/dot-analytics-search.component.ts

```diff
@@ -35,7 +35,7 @@ import { ANALYTICS_MONACO_EDITOR_OPTIONS, ANALYTICS_RESULTS_MONACO_EDITOR_OPTION
     templateUrl: './dot-analytics-search.component.html',
     styleUrl: './dot-analytics-search.component.scss'
 })
-export class DotAnalyticsSearchComponent {
+export default class DotAnalyticsSearchComponent {
```

Comment:
Why default?

Reply:
For use directly `loadComponent: () => import('./lib/dot-analytics-search/dot-analytics-search.component')` witout `.then(m => m.myCompoent)`